app/editPlan/get.controller.js

```diff
@@ -1,5 +1,22 @@
-const editPlan = async ({ params: { id }, session: { 'x-auth-token': token } }, res) => {
-  res.render(`${__dirname}/index`, { id, token })
+const { getSentencePlan } = require('../../common/data/sentencePlanningApi')
+const { getAboutTheIndividual, getAddObjectives, getFinalInformation } = require('./getTaskListData')
+
+const editPlan = async ({ path, params: { id, planId }, session: { 'x-auth-token': token } }, res) => {
+  try {
+    const sentencePlan = await getSentencePlan(planId, token)
+    const stub = path
+    const objectivesSection = getAddObjectives(sentencePlan, stub)
```

Comment:
Should this be bundled in with the other sections, on line 12?

Reply:
getFinalInformation needs to know if the objectives are complete, so I had to define it before so I could reference the results. Its a little bit untidy, but the best I could do.